src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -252,12 +259,18 @@
     <PackageReference Include="dotNetRDF">
       <Version>1.0.8.3533</Version>
     </PackageReference>
-    <PackageReference Include="NuGet.Packaging">
-      <Version>5.0.0-preview1.5665</Version>
+    <PackageReference Include="NuGet.Services.Entities">
+      <Version>2.41.0</Version>
```

Comment:
NuGet.Services.Entities is not 4.4.X versions since it is made in gallery. We should be pulling in latest of these guys.

Reply:
The latest version should be `4.4.5`, right?

Reply:
AFAICT the latest version of NuGet.Services.Entities is `2.41.0`.